lib/fastlane/plugin/wpmreleasetoolkit/actions/common/upload_to_s3.rb

```diff
@@ -111,11 +125,24 @@ def self.available_options
           ),
           FastlaneCore::ConfigItem.new(
             key: :skip_if_exists,
-            description: 'If the file already exists in the S3 bucket, skip the upload (and report it in the logs), instead of failing with `user_error!`',
+            description: '[DEPRECATED: Use if_exists instead]. If the file already exists in the S3 bucket, skip the upload (and report it in the logs), instead of failing with `user_error!`. When if_exists is set, this option is ignored',
             optional: true,
             default_value: false,
             type: Boolean
           ),
+          FastlaneCore::ConfigItem.new(
+            key: :if_exists,
+            description: 'What do to if the file file already exists in the S3 bucket. Possible values :skip, :replace, :fail. When set, overrides the deprecated skip_if_exists option',
+            optional: true,
+            is_string: false,
```

Comment:
Given [the auto-convert mechanism baked into `ConfigItem`](https://github.com/fastlane/fastlane/blob/master/fastlane_core/lib/fastlane_core/configuration/config_item.rb#L278), you can use `Symbol` as a type and it will auto-convert strings to symbols for you (see my reply on your other comment).
```suggestion
            type: Symbol,
```
(And then you can remove the check in your `verify_block` about the value responding to `:to_sym`, to only keep the check about accepted values)